cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -1978,7 +1979,7 @@ private synchronized X509TrustManager getDelegate() {
                         theTrustStore.load(null);
                     }
                     for (X509Certificate current : temporarilyTrusted) {
-                        theTrustStore.setCertificateEntry(current.getSubjectX500Principal().getName(), current);
+                        theTrustStore.setCertificateEntry(UUID.randomUUID().toString(), current);
```

Comment:
Are these ever cleaned up?

Reply:
They are not persisted, they are just added to the key-store instance but not stored. 
So they are present until a new delegate is created.